main.go

```diff
@@ -231,3 +247,22 @@ func max(i, j int) int {
 	}
 	return j
 }
+
+// fmtEtimeDuration formats etime's duration based on ps' format:
```

Comment:
Can you add a link where the format is specified?

Reply:
added it below but Github still shows this suggestion as active.